src/background-plugin.ts

```diff
@@ -544,6 +562,15 @@ class HttpContext {
 
     this.lastStatusCode = response.status;
 
+    if (this.lastHeaders !== null){
```

Comment:
We can use `this.lastHeaders = Object.fromEntries(response.headers)` here – `response.headers` is an instance of `Headers`, which should take care of the normalization of the header casing.